apps/www/styles/index.css

```diff
@@ -85,14 +85,20 @@ h6:not(.overwrite) {
 }
 
 .toc-animate {
-  transform: translate3d(3px, 0, 0) !important;
-  color: hsla(153, 60%, 53%, 1) !important;
+  @apply !translate-x-1 !text-brand-900;
 }
 
 .prose-toc a {
   transition: all 0.07s ease-in-out;
 }
 
+.quote-figure p:first-child {
```

Comment:
this can be put in tailwind config

Reply:
in the typography setting. then the same styling can be used across all the apps.

Reply:
https://github.com/supabase/supabase/blob/0a020247728b8a82a03115f4fba01bfede871bcd/packages/config/tailwind.config.js#L29-L164

Reply:
This doesn't seem to work though... can't use class selectors here? @MildTomato 

<img width="430" alt="Screenshot 2023-04-13 at 15 47 56" src="https://user-images.githubusercontent.com/25671831/231779122-09087ab1-482a-4b38-a3ea-654451d1b892.png">

Reply:
you did restart local ? you have there's no hot reloading of tailwind config

Reply:
I thought I did, but I didn't! 😁